### PR TITLE
feat(Dasboards): improving the dashboards pages

### PR DESCRIPTION
**Pull request by nbaenam**
Updating the [Introduction to dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and [Manage your dashboard](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/) pages.

See [NR-76731](https://issues.newrelic.com/browse/NR-76731).

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 1]
```diff
@@ -65,89 +67,93 @@ import dashboardsTimePicker from 'images/dashboards_screenshot-crop_time-picker.
...
+    
+    The dashboard is automatically copied and the duplicate is added to the [index](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#get-started). The duplicate has [**Public - Read and write** permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions). 
+    
+  </Collapser>
 <Collapser
     id="dash-edit"
     title="Edit your dashboard"
   >
-    You can edit a dashboard if you have [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
+    You can edit a dashboard if you've [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
```
Comment (homelessbirds) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:104]: I could be the odd one out here, but I feel like "you have" if one of the few contractions we should refrain from using. It feels a little awkward to me. [Here's](https://www.oxfordlearnersdictionaries.com/us/definition/english/you-ve#:~:text=%E2%80%8Byou%20have%20You've,Questions%20about%20grammar%20and%20vocabulary%3F ) some food for thought.

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx [hunk 2]
```diff
@@ -65,89 +67,93 @@ import dashboardsTimePicker from 'images/dashboards_screenshot-crop_time-picker.
...
 
-      * When viewing a dashboard, select the <Icon name="fe-tv"/> icon at the top right.
-
-      * Add this parameter to a dashboard page URL: `&platform[tvMode]=true`
-
-
-  To configure TV mode, from a dashboard, click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Settings**. Under the TV mode section, there're these options:
-
-      * **Display the name of the dashboard**. Turning off the dashboard name gives the dashboard charts more space on the screen.
+    Exporting to PDF or as an image has certain limitations depending on the chart type.
```
Comment (homelessbirds) [inline src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:120]: What if you wrote something more like:

Exporting to PDF or as an image changes the display depending on the chart type.

**Comment by homelessbirds** [comment]
I just have two small suggestions. Otherwise, this looks great!

**Review by homelessbirds — APPROVED** [review]

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:104]: Done!

File: src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx:120]: Much better!!! Thank you, @homelessbirds!

**Comment by nbaenam** [comment]
I merge it because Sammie already reviewed and I only added her comments.

**Pull request merged**